README.md

```diff
@@ -87,11 +87,11 @@ Before beginning the program, apprentices meet weekly for 4 weeks to learn JavaS
 1. [Debugging](/debugging/debugging.md)
 
 ### Week 3 - JavaScript Fundamentals
-1. [JavaScript 1](/javascript/javascript-1.md)
-1. [JavaScript 2](/javascript/javascript-2.md)
-1. [JavaScript 3](/javascript/javascript-3.md)
-1. [JavaScript 4](/javascript/javascript-4.md)
-1. [JavaScript 5](/javascript/javascript-5.md)
+1. [JavaScript 1 - Variables, Strings, Numbers](/javascript/javascript-1-variables.md)
+1. [JavaScript 2 - Array Functions](/javascript/javascript-2-array-functions.md)
```

Comment:
```suggestion
1. [JavaScript 2 - Arrays, Functions](/javascript/javascript-2-arrays-functions.md)
```

It's about arrays and functions, not "array functions"

Reply:
also, please change all other instances accordingly 😊

Reply:
On it. I'm making the changes @alodahl